### DISCUSSION TITLE
Problem Evergreen wallet

### DISCUSSION THREAD

**Discussion by Mikael2121** [Chia-Network/chia-blockchain#17181] [Support]
two transactions are confirmed  from Exchange and visible in chiascan, but the balance on my Evergreen wallet shows zero

**Comment by BrandtH22** [comment]
Hey @Mikael2121 , this sounds like a syncing issue with the wallet but unfortunately the evergreen app was developed and is managed by a third party so you will need to work with their team to determine what might be the cause.

Here is the evergreen discord server (https://discord.gg/sAZTy4DZJ7)

If you decide to use the official desktop chia wallet (https://www.chia.net/downloads/) then we can provide more support by reopening this ticket here or by joining us in our discord server (https://discord.gg/chia).
